### DISCUSSION TITLE
My account got hacked ! please help me

### DISCUSSION THREAD

**Discussion by ghost** [community/community#110755] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

My account got hacked, and hacker either deleted my account or changed my username ,which was pkumars397. and unable to look up on my account,and also he changed my username too ,i am in depression ,please help me ,how to recover it,neither github support team is replying me via my new account,which i created to report this and get my account back.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @pk397sad, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
